Fix "line 579: undefined symbol: UL_SCNsA" problem at compile time in
lib/colors.c

Problem previously reported at:
http://www.spinics.net/lists/util-linux-ng/msg10496.html

Change taken from upstream git repo:
https://github.com/karelzak/util-linux/blob/master/lib/colors.c

--- util-linux-2.25.2/lib/colors.c.orig	2015-02-16 16:36:01.215073454 -0800
+++ util-linux-2.25.2/lib/colors.c	2015-03-01 10:53:21.301521771 -0800
@@ -576,9 +577,7 @@
 		if (*p == '\0' || *p == '#')
 			continue;
 
-		rc = sscanf(p,  UL_SCNsA" "	/* name */
-				UL_SCNsA,	/* color */
-				&cn, &seq);
+		rc = sscanf(p, "%128[^ ] %128[^\n ]", cn, seq);
 		if (rc == 2 && cn && seq)
 			rc = colors_add_scheme(cc, cn, seq);	/* set rc=0 on success */
 		if (rc) {
